src/main/java/codeu/controller/ChatServlet.java

```diff
@@ -148,15 +148,22 @@ public void doGet(HttpServletRequest request, HttpServletResponse response)
       response.getWriter().write(json);
       return;
     }
+    List<UUID> userSubbedList = user.getSubscriptions();
+    Boolean subValue = false;
```

Comment:
nit: These five lines can just be `Boolean subValue = user.getSubscriptions().contains(conversationId);`